apps/site-landing/src/containers/HelpDocs/FaqItem.tsx

```diff
@@ -1,10 +1,10 @@
 import React from "react";
 import { FAQDocument } from "components/Prismic/types";
 import { Header } from "semantic-ui-react";
-import { Renderer } from "./Renderer/Renderer";
+import { RichText, RichTextBlock } from "prismic-reactjs";
 
 export const FaqItem = ({ data }: FAQDocument) =>
   <div className={"x6spaceBefore x6spaceAfter"}>
-    <Header as={"h3"}><Renderer r={data.question} /></Header>
-    <Renderer r={data.answer} />
+    <Header as={"h3"}>{data.question ? data.question[0].text : ""}</Header>
```

Comment:
We don't want people to be able to change the size of the title that would break the design